src/cljs/main/broadfcui/components/Collapse.cljs

```diff
@@ -0,0 +1,26 @@
+(ns broadfcui.components.collapse
+  (:require
+    [dmohs.react :as react]
+    [broadfcui.common.icons :as icons]
+    [broadfcui.utils :as utils]
+    ))
+
+(react/defc Collapse
```

Comment:
FYI: For a while I was trying to avoid the redundancy of having a component name duplicate its namespace. For example, I preferred `workspace/Page` over `workspace/WorkspacePage`. In practice, I don't think this redundancy causes any harm and it's nicer to have a semi-unique component name. In other words, I like this pattern.